index.html

```diff
@@ -44,7 +44,7 @@
     <div class="shopping-cart">
       <p class="cart-title">Items in Cart</p>
       <div class="cart-list"></div>
-      <p>Total: $<span class="total">0</span></p>
+      <p style="font-weight: bold;">Total: $<span class="total">0</span></p>
```

Comment:
Ok, I'll prevent from using it in the future, thanks!

On Wed, Mar 30, 2016 at 6:53 PM, Aaron Hayslip <notifications@github.com>
wrote:

> In index.html
> <https://github.com/Elevationacademy/shopping-cart/pull/5#discussion_r57913487>
> :
>
> > @@ -44,7 +44,7 @@
> >      <div class="shopping-cart">
> >        <p class="cart-title">Items in Cart</p>
> >        <div class="cart-list"></div>
> > -      <p>Total: $<span class="total">0</span></p>
> > +      <p style="font-weight: bold;">Total: $<span class="total">0</span></p>
>
> It's not terrible in this case, but in general inline styles are a bad
> idea because they're very hard to override.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Elevationacademy/shopping-cart/pull/5/files/6587ca0bdc6ed100806fcd623fc31e56877b0a54#r57913487>
>